packages/create-v2-addon-repo/src/blueprints/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "pnpm": {
     "overrides": {
+      "@ember/test-waiters": "^4.0.0",
```

Comment:
`ember-a11y-testing` needs to widen its support.

https://github.com/ember-a11y/ember-a11y-testing/blob/v7.1.0/package.json#L65